README.md

```diff
@@ -45,6 +45,18 @@ Repository for TDR transfer code
 
 * run  `npm install` then `npm run build`
 
+### Generated GraphQL classes
+There is a separate repository which contains the generated case classes needed to query the consignment API. 
+These classes will be needed by more than one project which is why they are in a separate project.
+If you need to add a new query:
+
+* Run `git clone https://github.com/nationalarchives/tdr-generated-graphql.git`
+* Add the new query to the `src/main/graphql` directory
+* Bump the build.sbt version by one patch version.
```

Comment:
Are we using semver for this? It's not a huge deal for an internal API for a small team, but it might help us keep track of backwards-incompatible changes.

Reply:
I've changed this project so it auto bumps the patch version. For larger breaking changes, we can manually release it with a minor or major version bump.

Reply:
The project will automatically bump the patch version for this. If we need to do anything else, we can specify a specific minor or major version manually but for most use cases, the automatic version bump will be fine.